charts/spire/charts/spire-server/values.yaml

```diff
@@ -206,23 +206,25 @@ nodeAttestor:
 
 # tornjak - Tornjak specific configuration
 tornjak:
+  # -- enable, to deploy Tornjak API (backend) component
   enabled: false
-  # image - Tornjak image (frontend + backend) if not separated above
-  image: # ghcr.io/spiffe/tornjak
+  image:
+    # registry: gcr.io
+    # repository: spiffe-io/tornjak-be
     registry: ghcr.io
-    repository: spiffe/tornjak
+    repository: spiffe/tornjak-be
     pullPolicy: IfNotPresent
-    # Overrides the image tag whose default is the chart appVersion.
-    # TODO we should use a specific Tornjak version instead of 'latest'
-    version: "latest"
+    # -- Overrides the image tag whose default is the chart appVersion.
```

Comment:
```suggestion
```

We can remove the comment for Tornjak as it doesn't apply. Tornjak can't use the chart version as spire is versioned differently.